code/modules/mob/living/carbon/superior_animal/superior_animal.dm

```diff
@@ -357,6 +365,12 @@
 		already_destroying_surroundings = TRUE
 	if(ranged)
 		stop_automated_movement = TRUE
+		if(get_dist(src, targetted_mob) <= comfy_range)
+			stance = HOSTILE_STANCE_ATTACKING
+			return //We do a safty return
```

Comment:
i dont think this addition works because this will 1. kill the switch, 2. stance is already set to this, 3. im also p sure it'll still jump to the 3rd proc

Reply:
er, wont jump

Reply:
tested to work